src/slack/constants.js

```diff
@@ -2,22 +2,74 @@ import * as vlConstants from './vacuumlabs/constants'
 import * as testConstants from './test/constants'
 import * as wincentConstants from './wincent/constants'
 
-export const MESSAGES = {
+const REASON = 'reason'
+const MANAGER = 'manager'
+
+const REASON_WITH_ADDRESS_QUESTION = {
+  [REASON]: ':question: Why do you need these items? Reply by sending a message. And don\'t forget to tell us your address!',
+}
+
+const REASON_DEFAULT_QUESTION = {
+  [REASON]: ':question: Why do you need these items? Reply by sending a message.',
+}
+
+const REASON_NOTE_QUESTION = {
+  [REASON]: ':pencil: Add a note by sending a message. And don\'t forget to tell us your address!',
+}
+
+const REASON_COMMENT_QUESTION = {
+  [REASON]: ':pencil: Send your comment in a message.',
+}
+
+const MANAGER_QUESTION = {[MANAGER]: ':question: Name of your manager (needed for approval for items above 100 EUR - write N/A otherwise). Reply by sending a message.'}
+
+const NOTIFICATION = {
+  accepted: 'Your order was accepted by office manager. You will be notified when the items get ordered.',
+  ordered: 'Your items were ordered. You will be notified when they arrive.',
+  delivered: 'Your order has arrived :truck: Come pick it up during office hours. If it was a personal order, please bring the money in CASH.',
+}
+
+const DEFAULT_MESSAGES = {
   home: {
-    company: ':question: Why do you need these items? Reply by sending a message. And don\'t forget to tell us your address!',
-    personal: ':pencil: Add a note by sending a message. And don\'t forget to tell us your address!',
+    company: {
+      ...REASON_WITH_ADDRESS_QUESTION,
+      ...MANAGER_QUESTION,
+    },
```

Comment:
it keeps troubling me that the app flow (how it asks for two inputs) here is guided by constants. it makes the already confusing code even more confusing. also the approach of having everything in constants so it can be reused, even with the `[REASON]:` part, it doesn't help code clarity either.

however, I don't have much better ideas now, or those would require bigger refactoring. I think we can merge it as is and have a call about potential refactoring of `slack.js` later. (I'm actually thinking about inlining parts of the `submitOrder` function for clarity on what happens when handling specific actions)

Reply:
I tried to avoid a bunch of repetitive code and reuse constants instead.
But as you said, it's better to make a potential refactoring of `slack.js` later.